test/SafexMain.js

```diff
@@ -0,0 +1,255 @@
+const { ethers } = require("hardhat");
+const { use, expect } = require("chai");
+const { solidity } = require("ethereum-waffle");
+
+use(solidity);
+use(require("chai-as-promised")).should();
+
+describe("SafexMain", async () => {
+  let autoAppealableArbitrator, safexMain, acc0, acc1, acc2, acc3, acc4, acc5, acc6, acc7, acc8, acc9;
```

Comment:
Instead of acc0, acc1 and so on, can we use only few addresses and rename them accordingly to who uses which account? So it wouldn't be confusing as to who uses which address.